CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing
+
+Contributions are **welcome** and will be fully **credited**.
+
+We accept contributions via Pull Requests on [Github](https://github.com/phpflo/phpflo).
+
+
+## Pull Requests
+
+- **[PSR-2 Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** - Check your code against the PSR-2 coding standard
+
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests and don't follow our unit tests coding standards.
+
+- **Document any change in behaviour** - Make sure the `README.md` and any other relevant documentation are kept up-to-date.
+
+- **Consider our release cycle** - We try to follow [SemVer v2.0.0](http://semver.org/). Randomly breaking public APIs is not an option.
+
+- **Create feature branches** - Don't ask us to pull from your master branch.
```

Comment:
I do not think this (feature branches) is necessary. The less words there are, the easier it is for people to read.